v2/test/incoming_data.json

```diff
@@ -0,0 +1 @@
+{"SurveyID (S)": "002", "UserID (S)": "12345678", "Age (S)": "11-30", "Calendar (M)": {"onset": {"S": "05/25/2020"}}, "Country (S)": "United States of America", "CountryCode (S)": "US", "Diagnosis (M)": {"tested": {"S": "no"}}, "Gender (S)": "male", "PostalCode (S)": "10001", "ReportDate (S)": "2020-05-26 12:00:00", "ReportSource (S)": "survey_app", "Symptoms (S)": {"cough": true, "fatigue": true}}
```

Comment:
Is this the sample that will come from App?? Might have to adjust the payload with what's in the api

Reply:
**_Indeed, is not correct._** Need to adjust. Could I ask that you paste an exact instance of what that payload might be **here** – ?